templates/deployment-zabbix-webservice.yaml

```diff
@@ -51,7 +51,11 @@ spec:
           {{- toYaml .Values.zabbixwebservice.resources | nindent 10 }}
         securityContext:
           {{- toYaml .Values.securityContext | nindent 12 }}
+        {{- if .Values.zabbixwebservice_image.tag }}
+        image: "{{ .Values.zabbixwebservice.image.repository }}:{{ .Values.zabbixwebservice_image.tag }}"
```

Comment:
Hi @scoopex!

Can you change

`image: "{{ .Values.zabbixwebservice.image.repository }}:{{ .Values.zabbixwebservice_image.tag }}"`

by

`image: "{{ .Values.zabbixwebservice.image.repository }}:{{ .Values.zabbixwebservice.image.tag }}"`

?

Reply:
Sure, done :-)